package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"name": "react-password-checklist",
 	"version": "1.5.0",
+	"type": "module",
```

Comment:
Looks like the Jest config is triggering failing builds with this change - I believe the `/jest.config.js` needs renamed to `/jes.config.cjs` in order to work.